eosdatadistribution/MDNS/__init__.py

```diff
@@ -39,7 +39,7 @@ def __init__(self):
         self.pretty_name = avahi.ServiceTypeDatabase.ServiceTypeDatabase()
 
     def get_human_type(self, type):
-        if self.pretty_name.has_key(type):
+        if type in self.pretty_name:
```

Comment:
[Seems to](https://github.com/lathiat/avahi/blob/master/avahi-python/avahi/ServiceTypeDatabase.py.in#L81)